ct/qbittorrent.sh

```diff
@@ -27,32 +27,24 @@ function update_script() {
     msg_error "No ${APP} Installation Found!"
     exit
   fi
-  if [[ ! -f /opt/${APP}_version.txt ]]; then
-    touch /opt/${APP}_version.txt
-    mkdir -p $HOME/.config/qBittorrent/
-    mkdir -p /opt/qbittorrent/
-    [ -d "/.config/qBittorrent" ] && mv /.config/qBittorrent "$HOME/.config/"
-    $STD apt-get remove --purge -y qbittorrent-nox
-    sed -i 's@ExecStart=/usr/bin/qbittorrent-nox@ExecStart=/opt/qbittorrent/qbittorrent-nox@g' /etc/systemd/system/qbittorrent-nox.service
-    systemctl daemon-reload
+  if [[ ! -f ~/.qbittorrent ]]; then
+    msg_error "Please create new qBittorrent LXC. Updating from v4.x to v5.x is not supported!"
```

Comment:
I'm having trouble updating from previous version of qBittorrent script.

I had an old LXC made from tteck's script, it was updated successfully to the latest version of qBittorrent (5.1.2) when you switched scripts to static builds.

But now I'm having this error which is a bit misleading, I'm having 5.1.2 but original script was installing https://packages.debian.org/bookworm/qbittorrent which is 4.5.2 and I think some directories changed.

Is there any way to update using latest script?